README.md

```diff
@@ -53,6 +53,7 @@ the resources of this module:
 - Healthcare DICOM Admin: `roles/healthcare.dicomStoreAdmin`
```

Comment:
Pin Terraform provider version here https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/versions.tf to be >= 3.49.0 as consent store is only available since that https://github.com/hashicorp/terraform-provider-google/releases/tag/v3.49.0

Reply:
google-beta should also be pinned in the same way